docs/openapi-qtsp.yaml

```diff
@@ -0,0 +1,185 @@
+openapi: 3.0.3
+
+info:
+  title: Firma con IO QTSP
+  description: Any APIs called by IO Backend to QTSP
+  version: 1.0.0
+
+paths:
+
+  /tos:
+    summary: Request terms of service
+    get:
+      operationId: GetTos
+      tags:
+        - QTSP
+
+      responses:
+        '200':
+          description: Terms of service to be accepted
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/TOS"
+        
+        default:
+          description: Generic error
+      
+      
+      
+  /login:
+    summary: Request login to QTSP
+    post:
+      operationId: GetChallenge
+      tags:
+        - QTSP
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Auth'
+      responses:
+        '200':
+          description: Challenge generated
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Challenge"
+        400:
+          description: Bad request
+        401:
+          description: SAML assertion not verified
+        404:
+          description: User not found
+        default:
+          description: Generic error
+      
+  /sign:
```

Comment:
`sign` is a verb, so there is the same problem as the `login` endpoint

this is the endpoint that actually makes the `signature` ?